tests/fakeapi/hello.py

```diff
@@ -151,7 +151,7 @@ def schema_response_integer(valid):
     if valid == "valid":
         return 3
     else:
-        return 3.0
+        return 3.14
```

Comment:
what's the background of this change?

Reply:
Seems that jsonschema draft 7 treats `3.0` as integer, because `3.0 == 3`.  
I havent search where/when such change has been made in jsonschema.

Reply:
Unit tests were failing without that update. And because it looks pretty resonable, I applied that.

Reply:
@hjacobs I've found more detailed explanation:
* https://github.com/Julian/jsonschema/issues/575#issuecomment-513679876
* https://json-schema.org/latest/json-schema-core.html#rfc.section.4.2.3